### PR TITLE
Added metadata properties

### PR DESCRIPTION
**Pull request by egauzens**
# Description

Added additional metadata to the dataset results to match the wireframes found here: https://app.abstract.com/share/ca9ac8ee-68b2-4422-b0d7-55a61c0bb403. Just a heads up, I ran the linter on the index file and it formatted a fair amount of extraneous stuff that I did not touch

## Type of change

Delete those that don't apply.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?
In order to run locally you must have the following env variables set:

ALGOLIA_API_KEY=9f55092cb37c3896a8bfc48ef87206d6
ALGOLIA_APP_ID=04WW1V1O0F

Ran changes locally and navigated to the 'Datasets' and 'Simulations' tabs to verify that metadata is displayed properly:

image

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: components/SearchResults/DatasetSearchResults.vue [hunk 1]
```diff
@@ -91,16 +97,40 @@ export default {
...
+      default: () => new Map()
     }
   },
 
   data() {
     return {
       sortOrders: ['ascending', 'descending'],
       PROPERTY_DATA: [
+        {
+          displayName: 'Anatomical Structure',
```
Comment (egauzens) [inline components/SearchResults/DatasetSearchResults.vue:112]: These display names will eventually come from contentful

**Review by Tehsurfer — COMMENTED** [review]
Is there a way to test this? I'd like to run a search with filters on the `/data` page.

I don't have the Algolia keys yet that are used in the env variables so unfortunately can't check how this is working locally.

**Comment by egauzens** [comment]
> Is there a way to test this? I'd like to run a search with filters on the `/data` page.
> 
> I don't have the Algolia keys yet that are used in the env variables so unfortunately can't check how this is working locally.

Oh sorry, yeah you will need to run the script that I put in the SPARC-Portal slack thread to have the algolia env variables set

**Comment by egauzens** [comment]
> > Is there a way to test this? I'd like to run a search with filters on the `/data` page.
> > I don't have the Algolia keys yet that are used in the env variables so unfortunately can't check how this is working locally.
> 
> Oh sorry, yeah you will need to run the script that I put in the SPARC-Portal slack thread to have the algolia env variables set. You can pull down the changes locally and run 'source setup_sparc.sh' in a command window

**Review by jwagenaar — APPROVED** [review]

**Pull request merged**